### PR TITLE
Rcon Web

### PR DESCRIPTION
**Pull request by eaglesemanation**
Chart for `itzg/rcon` deployment, implements #115

### CODE REVIEW

**Review by itzg — COMMENTED** [review]
This looks great! Feel free to mark the PR ready when you are.

**Comment by itzg** [comment]
Looks like the test setup needs to be tweaked

https://github.com/itzg/minecraft-server-charts/runs/5653804720?check_suite_focus=true#step:9:192

File: charts/rcon-web-admin/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+              value: {{ .Values.rconWeb.websocketRcon | default false | ternary "TRUE" "FALSE" | quote }}
+            {{- if .Values.ingress.enabled }}
+            {{ $wsUrl := print .Values.ingress.host (trimSuffix "/" .Values.ingress.path) "/websocket" }}
+            - name: RWA_WEBSOCKET_URL_SSL
+              value: {{ print "wss://" $wsUrl | quote }}
+            - name: RWA_WEBSOCKET_URL
+              value: {{ print "ws://" $wsUrl | quote }}
+            {{- end }}
+          {{- if not .Values.ingress.enabled }}
+          command:
```
Comment (eaglesemanation) [inline charts/rcon-web-admin/templates/deployment.yaml:73]: I don't really like this whole part, but there is unfortunate loop of "To deploy this chart you need to configure websocket url, to get websocket url you need to deploy this chart". Better solution would be to resolve it upstream, but for now this should work fine

Comment (itzg) [reply charts/rcon-web-admin/templates/deployment.yaml:73]: Understood and agreed that I don't like how rcon-web handles websockets in the first place with the requirement of a dedicated port.

**Comment by eaglesemanation** [comment]
Sorry, tried to recreate github workflows locally with `act`, but it kept breaking, so I had to spam small changes to see if they work.

**Comment by itzg** [comment]
No problem at all. I do the same.

File: charts/rcon-web-admin/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Out of the box, RCON Web Admin can check users for high ping, VAC status, or filter the chat for you around the clock.
+
+## Introduction
+
+This chart creates a single RCON Web Admin Pod, plus Services for HTTP and WebSocket traffic.
+
+## Prerequisites
+
+- 512 MB of RAM
+- Kubernetes 1.4+ with Beta APIs enabled
```
Comment (eaglesemanation) [inline charts/rcon-web-admin/README.md:16]: I only tested with v1.23.4+k3s1 and kind v1.23.1, so I have no clue which versions are supported :)

Comment (itzg) [reply charts/rcon-web-admin/README.md:16]: Yeah 1.4 seems ancient. I don't mind leaving it or going with 1.23 since that's confirmed.

**Review by itzg — APPROVED** [review]
All looks great!

**Pull request merged**